api/db/seeds/data/team-acces/build-sco-organizations.js

```diff
@@ -86,6 +92,28 @@ function _buildJosephineBaker(databaseBuilder) {
   ].forEach(_buildAdminMembership(databaseBuilder));
 }
 
+function _buildScoManagingStudentsNonGarOrganization(databaseBuilder) {
+  const organization = databaseBuilder.factory.buildOrganization({
+    id: SCO_NO_GAR_ORGANIZATION_ID,
+    type: 'SCO',
+    name: 'Lycée Pa Gar',
```

Comment:
Pour favoriser la compréhension : 
```suggestion
    name: 'Lycée pas-GAR',
```

Reply:
Corrigé ici : https://github.com/1024pix/pix/pull/10096/commits/8fc51535fa5004cacff0ffa3a2846b6e2524093d